GSoC_resources.md

```diff
@@ -6,67 +6,65 @@ permalink: gsoc_resources
 
 All of us go through the same process of devouring the internet when we are writing our proposal for GSoC. This is an attempt to bookmark all those resources in one place.
 
-## GSoC 2017 Ideas page
+Please fill this [GSoC accepted proposal form](https://forms.gle/xaoXmpEdeAJ9c9Fn9), so that it could help other potential GSoC candidates to write a more competent proposal.
 
+## This year selected proposals
+### GSoC 2019
```

Comment:
I would prefer the heading to be "Selected projects", no need to mention "this" and "previous" sections, as "this year" would be relevant only for about 4 months of the yr at best.

Reply:
Removed this year selected proposals